lib/spack/spack/test/cmd/view.py

```diff
@@ -24,7 +24,8 @@ def create_projection_file(tmpdir, projection):
     return projection_file
 
 
-@pytest.mark.parametrize('cmd', ['hardlink', 'symlink', 'hard', 'add'])
+@pytest.mark.parametrize('cmd', ['hardlink', 'symlink', 'hard', 'add',
+                                 'copy', 'relocate'])
```

Comment:
Can we rework the test to receive an additional parametrized value `is_link`:
```suggestion
@pytest.mark.parametrize('cmd,is_link', [
    ('hardlink', False), 
    ('symlink', True), 
    ('hard', False), 
    ('add', True),
    ('copy', False),
    ('relocate', False)
])
```
? I think that would be clearer than `(cmd in ('symlink', 'add'))`

Reply:
I used a different method that I felt was cleaner to make this clearer, let me know if that works for you.